google-cloud-spanner/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Release History
 
+### 1.9.0 / 2019-03-08
+
+* Spanner Batch DML.
+  * Add Transaction#batch_update.
+  * Add BatchUpdate.
+  * Add BatchUpdateError.
+  * Add SpannerClient#execute_batch_dml.
+* Support Protobuf 3.7.
```

Comment:
I don't think this line is needed. These changes weren't made to the files that are packaged in the gem.